### ISSUE TITLE
[stable/wordpress] Ingress wrong apiVersion

### ISSUE THREAD

**Issue by holmesb** [helm/charts#17727]
**Version of Helm and Kubernetes**:
2.14.3 & 1.15.3

**Which chart**:
[stable/wordpress]

**What happened**:
Ingress is wrong apiVersion for newer Kubernetes versions (>=1.14).  It fails to be created.  Changing `ingress.apiVersion` from `extensions/v1beta1` to `networking.k8s.io/v1beta1` allows ingress to be created.

**Anything else we need to know**:
Can either parameterise `apiVersion:` or [this helper ](https://github.com/helm/charts/blob/c2cdb7602266ed6fbb689193fa0ed109e0e03490/stable/minio/templates/_helpers.tpl#L68)can automate (I've not tested).

**Comment by carrodher** [comment]
According to the [kubernetes 1.16 changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#deprecations-and-removals):
> Ingress resources will no longer be served from `extensions/v1beta1` in v1.20. Migrate use to the `networking.k8s.io/v1beta1` API, available since v1.14. Existing persisted data can be retrieved via the `networking.k8s.io/v1beta1` API.

So unless you are using k8s v1.20 it should not fail. What is the exact error you are receiving?

Note that if the change in #17731 is merged, k8s <1.14 wouldn't work, so I think it is safe waiting until `ingress.apiVersion: extensions/v1beta1` is deprecated in v1.20. Of course, just in case the issues are not related to this topic, but according to the release notes, it should work on 1.15.3 without any issues; if it is not the case, we can reevaluate it.

**Comment by holmesb** [comment]
@carrodher, #17731 hard-codes `apiVersion`.  I'm not suggesting this.  That [changelog ](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#additional-notable-feature-updates)isn't clear:

> Continued deprecation of extensions/v1beta1, apps/v1beta1, and apps/v1beta2 APIs; these extensions will be retired in 1.16!

All I know is, if I change `apiVersion` to networking.k8s.io/v1beta1, the ingress is successfully created.  In any case, this needs to be done before `extensions/v1beta1` is retired.  I'll create a PR.

**Comment by carrodher** [comment]
You're right, it is not clear. In fact, you're using 1.15.3, it is not even >=1.16...
Anyway, this new PR looks good to me. Thanks!
